services/workflows-service/src/workflow/workflow.service.ts

```diff
@@ -983,8 +987,10 @@ export class WorkflowService {
     context: DefaultContextSchema,
     entityId: TEntityId,
   ): Promise<DefaultContextSchema> {
+    if (!context?.documents?.length) return context;
```

Comment:
Could throw a specific error in the future or handle no documents differently, for example log something or send an alert. For now without this return we get a crash from mapping.